src/client/components/utils/MarketFeatures.ts

```diff
@@ -0,0 +1,7 @@
+import { Market, useMarket } from './CurrentLocale'
+
+export const useCreditCheckInfo = () => {
+  const market = useMarket()
+
+  return [Market.No].includes(market)
+}
```

Comment:
I really like this approach!

I would even name the module "features.ts" -- market is in the end just one way we differentiate features :)

Reply:
Maybe `featureToggles.ts`?